docs/apis/mana_old.md

```diff
@@ -0,0 +1,236 @@
+# Mana APIs (deprecated)
```

Comment:
Let's remove this file, everything here is in the mana.md.

Reply:
ok. removed it